src/plugins/googledrive/server/services/operation-service.js

```diff
@@ -0,0 +1,148 @@
+'use strict';
+const path = require('path');
+const process = require('process');
+const { google } = require('googleapis');
+module.exports = ({ strapi }) => ({
+  get: async (ctx) => {
+    const SERVICE_ACCOUNT_PATH = path.join(
```

Comment:
This is referencing the config https://github.com/dev-launchers/strapiv4/pull/250/files#diff-4be71c3182a0de045ef56aec0e1aa8d5bf9075135637452164bda6e0cb015aa0R78?